openedx/core/djangoapps/util/testing.py

```diff
@@ -50,17 +52,27 @@ def setUp(self):
             discussion_topics={}
         )
 
+        seed_permissions_roles(self.course.id)
+
         self.staff_user = UserFactory.create(is_staff=True)
         self.alpha_user = UserFactory.create()
         self.beta_user = UserFactory.create()
         self.non_cohorted_user = UserFactory.create()
-        for user in [self.staff_user, self.alpha_user, self.beta_user, self.non_cohorted_user]:
+        self.community_ta = UserFactory.create(username="community_ta")
+        self.community_ta.roles.add(Role.objects.get(name="Community TA", course_id=self.course.id))
+        for user in [
+            self.staff_user,
+            self.alpha_user,
+            self.beta_user,
+            self.non_cohorted_user,
+            self.community_ta
+        ]:
             CourseEnrollmentFactory.create(user=user, course_id=self.course.id)
 
         alpha_cohort = CohortFactory(
             course_id=self.course.id,
             name='Cohort Alpha',
-            users=[self.alpha_user]
+            users=[self.alpha_user, self.community_ta]
```

Comment:
You changed `alpha_cohort` users, doesn't this change the resulting tests in anyway?